ts/session/apis/seed_node_api/SeedNodeAPI.ts

```diff
@@ -146,9 +146,7 @@ export interface SnodeFromSeed {
 }
 
 const getSnodeListFromSeednodeOneAtAtime = async (seedNodes: Array<string>) =>
-  allowOnlyOneAtATime('getSnodeListFromSeednode', () =>
-    getSnodeListFromSeednode(seedNodes)
-  ) as Promise<Array<SnodeFromSeed>>;
+  allowOnlyOneAtATime('getSnodeListFromSeednode', () => getSnodeListFromSeednode(seedNodes));
```

Comment:
Same thing here

Reply:
it's a generic now and correctly typed ! thanks to https://type-level-typescript.com/